builders/integrations/relayers/gelato.md

```diff
@@ -36,278 +36,265 @@ Then, take the following steps:
 
-Next, you'll need to create a javascript file for your script. You can create a `hello-world.js` file by running:
+### Installation {: #installation }
+
+#### Gelato Relay SDK
+
+Gelato Relay SDK is an [NPM package](https://www.npmjs.com/package/@gelatonetwork/relay-sdk){target=\_blank} that can be installed locally with the following command:
 
 ```
-touch hello-world.js
+npm install @gelatonetwork/relay-sdk
 ```
 
-Now you're ready to build. First, you need to import the Gelato Relay SDK and EthersJS:
+or
 
 ```
-  import { Wallet, utils } from "ethers";
-  import { GelatoRelaySDK } from "@gelatonetwork/gelato-relay-sdk";
+yarn add @gelatonetwork/relay-sdk
 ```
 
-Then, create a function to contain the logic of the script:
+#### Gelato Relay Context
 
-```
-const forwardRequestExample = async () => {
+`GelatoRelayContext` is an [NPM package](https://www.npmjs.com/package/@gelatonetwork/relay-context){target=\_blank} which can be installed locally:
 
-}
+```
+npm install --save-dev @gelatonetwork/relay-context
 ```
 
-Within the `forwardRequestExample` function, define the chain ID and the [`HelloWorld.sol` contract](https://moonscan.io/address/0x3456E168d2D7271847808463D6D383D079Bd5Eaa#code){target=_blank} that you want to interact with.
+or
 
 ```
-  const chainId = {{ networks.moonbeam.chain_id }};
-  // `HelloWorld.sol` contract on Moonbeam
-  const target = "0x3456E168d2D7271847808463D6D383D079Bd5Eaa";
+yarn add -DE @gelatonetwork/relay-context
 ```
 
-The [`HelloWorld.sol` contract](https://moonscan.io/address/0x3456E168d2D7271847808463D6D383D079Bd5Eaa#code){target=_blank}, reproduced below, is configured to have gasless transaction support.
+If you would like your target contract to pay Gelato directly during the relay call forward, you can utilise your own logic (please make sure you have replay and re-entrancy protection and [whitelist GelatoRelay](https://docs.gelato.network/developer-products/gelato-relay-sdk/sdk-methods) for your target function) *or* you can inherit our ready-made `GelatoRelayContext` in your target contract. `GelatoRelayContext` is an extremely simple way to create a Gelato Relay compatible smart contract, with just one import. See more information [here](https://docs.gelato.network/developer-products/gelato-relay-sdk/prerequisites#gelatos-relay-context).
 
-```
-// SPDX-License-Identifier: MIT
-pragma solidity 0.8.13;
 
-import {ERC2771Context} from "@openzeppelin/contracts/metatx/ERC2771Context.sol";
+### Smart Contract Example
```

Comment:
Instead of Smart Contract Example, I recommend changing this to be more descriptive about what the example does like previously we had "Send a Gasless Transaction with Gelato Relay SDK"